Lib/test/test_tarfile.py

```diff
@@ -386,7 +386,7 @@ def test_null_tarfile(self):
     def test_ignore_zeros(self):
         # Test TarFile's ignore_zeros option.
         # generate 512 pseudorandom bytes
-        data = Random(0).getrandbits(512*8).to_bytes(512, 'big')
+        data = Random(0).randbytes(512)
```

Comment:
I think it is better to use the global `randbytes()`.

Reply:
It uses a seed of 0. I would prefer to use a temporary instance, rather than modifying random._inst.